README.md

```diff
@@ -6,9 +6,13 @@ Welcome! You are at the root of the Nava Labs Decision Support Tool pilot repo.
 
 To set up your local development environment, follow the instructions in [Getting Started](docs/app/getting-started.md).
 
+## Data Ingestion
+
+The data streams are currently separated by chat engines which are defined in [app/src/chat_engine.py](https://github.com/navapbc/labs-decision-support-tool/blob/main/app/src/chat_engine.py). To add a new engine, a class must be created with the following attributes engine_id,name, datasets (which must match the name of the ingestion dataset’s script, see below), and a formatter which formats the chat engine’s response.
```

Comment:
```suggestion
The data streams are currently separated by chat engines which are defined in [app/src/chat_engine.py](https://github.com/navapbc/labs-decision-support-tool/blob/main/app/src/chat_engine.py). To add a new engine, a class must be created with the following attributes: `engine_id`, `name`, `datasets` (which must match the name of the ingestion dataset’s script, see below), and a formatter which formats the chat engine’s response.
```

Reply:
Not quite right. The data sources are not coupled to any one chat engine, i.e. a chat engine can query any subset of data sources, which is set via `datasets`. Stating "data streams are currently separated by chat engines" sounds misleading to me since multiple engines can use the same dataset (data source). For this ingestion section, I would mention a "chat engine" only in the context of it being a downstream consumer of the data source.

Reply:
Revised the `Data Ingestion` content, @yoomlam  let me know if this clears up any confusion.